### ISSUE TITLE
Identity Dashboard FE - Setup eslint rules

### ISSUE THREAD

**Issue by acald-creator** [department-of-veterans-affairs/va.gov-team#58962]
This issue is to configure the proper ESlint rules for this project.

- `@department-of-veterans-affairs/recommended`

Next.js 13 also has its own ESlint configuration that comprises several rules as one, but since we're using `@department-of-veterans-affairs/recommended`, I removed `next/core-web-vitals` and used `next` as the last import.

Secondly, there needs to be another PR in the `../veteran-facing-services-tools/packages/eslint-plugin` to update `babel-eslint` to use `@babel/eslint-parser`. The package `babel-eslint` has been deprecated and is recommended to use `@babel/eslint-parser` instead.

I will be submitting a PR in `veteran-facing-services-tools/packages/eslint-plugin`.

**Comment by acald-creator** [comment]
Open PR on setting up ESlint rules to use `@department-of-veterans-affairs/recommended`.

https://github.com/department-of-veterans-affairs/identity-dashboard-fe/pull/5

**Comment by acald-creator** [comment]
Still waiting on PR approval on https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/pull/931

**Comment by acald-creator** [comment]
The PR has been updated frequently, but there are some rules that we will need to satisfy before emerging this PR in.
